packages/core/react/src/__tests__/WalletProviderDesktop-test.tsx

```diff
@@ -224,7 +224,7 @@ describe('WalletProvider when the environment is `DESKTOP_WEB`', () => {
             it('calls `autoConnect`', () => {
                 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                 const adapter = ref.current!.getWalletContextState().wallet!.adapter;
-                expect(adapter.autoConnect).toHaveBeenCalled();
+                expect(adapter.connect).toHaveBeenCalled();
```

Comment:
I'm not 100% sure about this, but does it make sense we test here (or any of the other places) for both the condition we want and don't want?

```suggestion
                expect(adapter.connect).toHaveBeenCalled();
                expect(adapter.autoConnect).not.toHaveBeenCalled();
```

Reply:
Yep agree with this! I've been making kinda ad-hoc changes to the existing tests, but makes sense to check both cases consistently.